knyst/examples/basic/modulation.rs

```diff
@@ -0,0 +1,65 @@
+use anyhow::Result;
+#[allow(unused)]
+use knyst::{
+    audio_backend::{CpalBackend, CpalBackendOptions, JackBackend},
+    controller::print_error_handler,
+    prelude::*,
+};
+
+/// Initializes the audio backend, sets up the `KnystSphere`, and configures a wavetable oscillator to output audio.
+///
+/// The function performs the following steps:
+/// 1. Initializes the audio backend and starts a `KnystSphere`.
+/// 2. Sets the active sphere for audio processing.
```

Comment:
The active sphere is not set explicitly, no need to document it here I don't think.